openpype/pipeline/create/utils.py

```diff
@@ -0,0 +1,122 @@
+import collections
+
+from openpype.client import get_assets, get_subsets, get_last_versions
+
+
+def get_last_versions_for_instances(
+    project_name, instances, use_value_for_missing=False
+):
+    """Get last versions for instances by their asset and subset name.
+
+    Args:
+        project_name (str): Project name.
+        instances (list[CreatedInstance]): Instances to get next versions for.
+        use_value_for_missing (Optional[bool]): Missing values are replaced
+            with negative value if True. Otherwise None is used. -2 is used
+            for instances without filled asset or subset name. -1 is used
+            for missing entities.
+
+    Returns:
+        dict[str, Union[int, None]]: Last versions by instance id.
+    """
+
+    output = {
+        instance.id: -1 if use_value_for_missing else None
```

Comment:
Here we are assuming all of them are missing, right? But later on I can't see where do we check for missing entities? So it could be that we return an instance with `-1` that has entities? i.e. Line 50